system/services/presideObjects/PresideObjectService.cfc

```diff
@@ -1081,6 +1083,14 @@ component displayName="Preside Object Service" {
 		);
 
 		for (var record in records ) {
+			var formName = record.configurator__formname ?: "";
```

Comment:
I'd like to remove this logic if possible. Is the purpose to ensure that checkbox boolean fields have a present variable? I think this logic is the responsibility of the code that prepares the json (i.e. all the required fields should be present and its not the responsibility of the preside object service to start validating the input).

I think this should be safe to remove altogether (along with the line that gets the forms service). Or is this logic absolutely required?